Fix bug with creating and editing habit events

MacMalainey opened the pull request:
<!-- Description of PR -->

## Description of Changes

<!-- Provide a list of changes here -->

## Screenshots

![Screenshot from 2021-11-05 13-07-26](https://user-images.githubusercontent.com/14865456/140565223-b8bb313f-e5c5-4a70-84f9-098e16633ae3.png)


## Checklist

- [ ] Automated tests
- [ ] Screenshots included (if necessary)
- [ ] Code is well commented

Closes #ISSUE_ID

j-vent reviewed (dismissed):
Add to ViewHabitEventActivity:
 @ Override
    public void onBackPressed(){
        // return back to main habit list
        Intent returnIntent = new Intent(getApplicationContext(), MainActivity.class);
        startActivity(returnIntent);
    }

MacMalainey commented:
Updated style and fixed broken tests
![image](https://user-images.githubusercontent.com/14865456/140569459-f638f610-4fe7-45e8-88d2-bfef9351c08b.png)

j-vent reviewed (approved):
LGTM
